scripts/main/sidebar.js

```diff
@@ -400,12 +466,8 @@ sidebar.createStructure.album = function (album) {
 		structure.basics.rows.push({ title: lychee.locale["ALBUM_SHOW_TAGS"], kind: "showtags", value: data.show_tags, editable });
 	}
 
-	let videoCount = 0;
-	$.each(data.photos, function () {
-		if (this.type && this.type.indexOf("video") > -1) {
-			videoCount++;
-		}
-	});
+	const videoCount = data.photos.reduce((count, photo) => (count + (photo.type.indexOf("video") > -1) ? 1 : 0), 0);
```

Comment:
```suggestion
	const videoCount = data.photos.reduce((count, photo) => (count + ((photo.type.indexOf("video") > -1) ? 1 : 0)), 0);
```

`+` has a higher precedence than ternary, so it would never go above `1`...

Reply:
I am confused. There has already been one pair of parentheses around the ternary, we do you want to add a second pair at the exact same position?

Reply:
Look closer -- the existing parentheses are not around the ternary but the condition :smiley:.

I only found this during testing -- `videoCount` never goes above `1`.

Reply:
I was confused by the change highlighting. I am completely overworked. I needed 4 commits for https://github.com/LycheeOrg/Lychee/pull/1275 :woozy_face:

Reply:
Now happened what I feared. I even thought I had added these parentheses once and then the prettier removed them. After I applied your suggestion, the prettier is whining. :-(

Reply:
LOL. Prettier is not aware of operator precedence?

Reply:
No, it's fine. It removed another spurious set of parethesis, but kept the one that matters.